NuKeeper.Github/Mappings/Account.cs

```diff
@@ -0,0 +1,20 @@
+using NuKeeper.Abstract;
+using Octokit;
+using System;
+#pragma warning disable CA1054 // Uri parameters should not be strings
+namespace NuKeeper.Github.Mappings
+{
+    public class GithubAccount : User, IAccount
```

Comment:
Something seems off creating interfaces for DTOs, are they really needed? It seems like base classes would work just fine here.

Reply:
I strongly second this comment.  Interfaces are used so that services can be a) injected by typed by the container and b) have behaviour mocked out for testing.  

But objects that are purely data not code are DTOs. DTOs have no behaviour to mock and can be created in tests when needed. They should not have interfaces.

Reply:
I have left these as interfaces at the moment. Happy to swap them to base classes though. let me know your thoughts